wurst/lib/ItemRecipes.wurst

```diff
@@ -0,0 +1,1431 @@
+
+    recipe = new ItemRecipe(ITEM_NETS, 3, true, true, true)
+        ..addItem(ITEM_TINDER)
+        ..addItem(ITEM_TINDER)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_SMOKE_BOMB, 3, true, false, true)
+        ..addItem(ITEM_ACORN)
+        ..addItem(ITEM_SPIRIT_WIND)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_DARK_THISTLES, 3, true, false, true)
+        ..addItem(ITEM_THISTLES)
+        ..addItem(ITEM_DARK_ROCK)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_TRANSPORT_SHIP_KIT, 1, true, true, true)
+        ..addItem(ITEM_STICK)
+        ..addItem(ITEM_STICK)
+        ..addItem(ITEM_CLAY_BALL)
+        ..addItem(ITEM_CLAY_BALL)
+        ..startBatch()
+            ..addItem(ITEM_ELK_SKIN_COAT)
+            ..addItem(ITEM_WOLF_SKIN_COAT)
+            ..addItem(ITEM_BEAR_SKIN_COAT)
+        ..endBatch()
+        ..startBatch()
+            ..addItem(ITEM_ELK_SKIN_COAT)
+            ..addItem(ITEM_WOLF_SKIN_COAT)
+            ..addItem(ITEM_BEAR_SKIN_COAT)
+        ..endBatch()
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_BATTLE_ARMOR, 0, true, false, true)
+        ..addItem(ITEM_BONE_COAT)
+        ..addItem(ITEM_MANA_CRYSTAL)
+        ..addItem(ITEM_SPIRIT_WIND)
+        ..addItem(ITEM_SPIRIT_WATER)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_ANABOLIC_BOOTS, 0, true, false, true)
+        ..addItem(ITEM_BONE_BOOTS)
+        ..addItem(ITEM_SPIRIT_WIND)
+        ..addItem(ITEM_SPIRIT_WATER)
+        ..addItem(ITEM_ANABOLIC_POTION, true, 1)
+        ..addItem(ITEM_ANABOLIC_POTION, true, 1)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_ANABOLIC_BOOTS, 0, true, false, true)
+        ..addItem(ITEM_BONE_BOOTS)
+        ..addItem(ITEM_SPIRIT_WIND)
+        ..addItem(ITEM_SPIRIT_WATER)
+        ..addItem(ITEM_ANABOLIC_POTION, true, 2)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_BATTLE_GLOVES, 0, true, false, true)
+        ..addItem(ITEM_BONE_GLOVES)
+        ..addItem(ITEM_STEEL_INGOT)
+        ..addItem(ITEM_STEEL_INGOT)
+        ..addItem(ITEM_MANA_CRYSTAL)
+        ..addItem(ITEM_SPIRIT_WIND)
+        ..addItem(ITEM_SPIRIT_WATER)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_EMP, 1, true, false, true)
+        ..addItem(ITEM_MANA_CRYSTAL)
+        ..addItem(ITEM_MANA_CRYSTAL)
+        ..addItem(ITEM_SPIRIT_WATER)
+        ..addItem(ITEM_SPIRIT_WIND)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_BATTLE_SHIELD, 0, true, false, true)
+        ..addItem(ITEM_BONE_SHIELD)
+        ..addItem(ITEM_ELK_HIDE)
+        ..addItem(ITEM_STEEL_INGOT)
+        ..addItem(ITEM_SPIRIT_WIND)
+        ..addItem(ITEM_SPIRIT_WATER)
+        ..addItem(ITEM_MANA_CRYSTAL)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_FIRE_BOMB, 2, true, false, true)
+        ..addItem(ITEM_SMOKE_BOMB)
```

Comment:
What if `SMOKE_BOMB` has 1 charge?

Reply:
The current recipe does not take into account smoke bomb charges either, it produces 2 fire bombs regardless.

Reply:
Precisely. For the rest of conflicts, I'll be planning ItemRecipe update for today to fix them.